wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -8468,7 +8476,6 @@
 		F1B2422E219B2C9B00F7035C /* Configuration */ = {
 			isa = PBXGroup;
 			children = (
-				A9B405DF269C812700ACE28C /* ios.xcconfig */,
```

Comment:
Removes this reference.

<img width="246" alt="Screenshot 2024-03-27 at 12 24 59" src="https://github.com/wireapp/wire-ios/assets/15628584/86bb5a82-de07-4718-81f5-2ad76ef111ec">